jms/src/test/scala/akka/stream/alpakka/jms/scaladsl/JmsConnectorsSpec.scala

```diff
@@ -304,6 +349,62 @@ class JmsConnectorsSpec extends JmsSpec {
       result.failed.futureValue shouldBe an[JMSException]
     }
 
+    "publish and consume elements through a topic with custom topic creator" in withServer() { ctx =>
+      import system.dispatcher
+
+      def createTopic(destinationName: String): Session => javax.jms.Topic = { (session: Session) =>
+        val amqSession = session.asInstanceOf[ActiveMQSession]
+        amqSession.createTopic(s"my-$destinationName")
+      }
+
+      val connectionFactory = new ActiveMQConnectionFactory(ctx.url)
+
+      //#create-topic-sink
```

Comment:
Don't duplicate any `//#` markers by copy-and-paste, use unique references in the docs.

Reply:
Done